specifications/scalar-universe-3.1.0.json

```diff
@@ -0,0 +1,651 @@
+        "content": {
+          "application/json": {
+            "schema": {
+              "$ref": "#/components/schemas/Error"
+            }
+          }
+        }
+      },
+      "Forbidden": {
+        "description": "Forbidden",
+        "content": {
+          "application/json": {
+            "schema": {
+              "$ref": "#/components/schemas/Error"
+            }
+          }
+        }
+      },
+      "NotFound": {
+        "description": "NotFound",
+        "content": {
+          "application/json": {
+            "schema": {
+              "$ref": "#/components/schemas/Error"
+            }
+          }
+        }
+      }
+    },
+    "schemas": {
+      "NewUser": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string",
+            "required": true,
+            "example": "Marc"
+          },
+          "email": {
+            "type": "string",
+            "format": "email",
+            "required": true,
+            "example": "marc@scalar.com"
+          },
+          "password": {
+            "type": "string",
+            "required": true,
+            "minLength": 8,
+            "example": "i-love-scalar"
+          }
+        }
+      },
+      "User": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer",
+            "format": "int64",
+            "required": true,
+            "example": 1
+          },
+          "name": {
+            "type": "string",
+            "required": true,
+            "example": "Marc"
+          },
+          "email": {
+            "type": "string",
+            "format": "email",
+            "required": true,
+            "example": "marc@scalar.com"
+          }
+        }
+      },
+      "Credentials": {
+        "type": "object",
+        "properties": {
+          "email": {
+            "type": "string",
+            "format": "email",
+            "required": true,
+            "example": "marc@scalar.com"
+          },
+          "password": {
+            "type": "string",
+            "required": true,
+            "example": "i-love-scalar"
+          }
+        }
+      },
+      "Token": {
+        "type": "object",
+        "properties": {
+          "token": {
+            "type": "string",
+            "example": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
+          }
+        }
+      },
+      "Planet": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "integer",
+            "format": "int64",
+            "required": true,
+            "example": 1
+          },
+          "name": {
+            "type": "string",
+            "required": true,
+            "example": "Mars"
+          },
+          "description": {
+            "type": ["string", "nullable"],
```

Comment:
I think we also support `"nullable": true", I'm not sure which way is the right way though

Reply:
the array is the recommended modern syntax, but we should add the old one, which is still supported too 👍